org-roam-protocol.el

```diff
@@ -5,7 +5,7 @@
 ;; URL: https://github.com/org-roam/org-roam
 ;; Keywords: org-mode, roam, convenience
 ;; Version: 1.2.1
-;; Package-Requires: ((emacs "26.1") (org "9.3"))
+;; Package-Requires: ((emacs "27.1") (org "9.3"))
```

Comment:
is this change necessary?

Reply:
It's because of these 

```
org-roam-protocol.el:73:16: error: You should depend on (emacs "27.1") if you need `org-link-make-string'.
org-roam-protocol.el:78:9: error: You should depend on (emacs "27.1") if you need `org-link-store-props'.
```

But I will make this more portable, I just did the minimum to get the PR green enough to make it worth your time :)

Reply:
You may make use of these functions for backward-compatibility:

https://github.com/org-roam/org-roam/blob/7722b65aaaff985bb9764746bc0a7a68877eaa7c/org-roam-compat.el#L85